config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   get "/healthcheck", to: proc { [200, {}, ["OK"]] }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get "/check-link", to: proc { [200, {}, [{}.to_json]] }
```

Comment:
I'm not sure about the need for `-link` here as that's fairly obvious from the service. Unless the plan for the batch endpoint is to be something like `/check-batch`?